packages/native-stack/src/views/HeaderConfig.tsx

```diff
@@ -73,16 +73,20 @@ export default function HeaderConfig({
 
   const [backTitleFontFamily, largeTitleFontFamily, titleFontFamily] =
     processFonts([
-      headerBackTitleStyleFlattened.fontFamily,
-      headerLargeTitleStyleFlattened.fontFamily,
-      headerTitleStyleFlattened.fontFamily,
+      headerBackTitleStyleFlattened.fontFamily ?? fonts.regular.fontFamily,
+      headerLargeTitleStyleFlattened.fontFamily ??
```

Comment:
`headerLargeTitleStyleFlattened` doesn't seem to be properly defaulted here for some reason

![image](https://user-images.githubusercontent.com/39658211/221599905-bfab2a0b-001f-4c4f-ada1-a92619461bfb.png)


`headerTitleStyleFlattened` is also not getting defaulted in the next line but it's defaulted in next few lines in `titleFontWeight` variable so the end result for small title is OK

Reply:
Fixed